changelog/13480.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed a few test failures in pytest's own test suite when run with ``-Wdefault`` or a similar override.
```

Comment:
FTR: since this is not end-user related, probably best as a "trivial" changelog entry, or even not worth having a changelog entry at all.